app/locales/en/common.json

```diff
@@ -0,0 +1,23 @@
+{
+  "Edit": "Edit",
+  "Undo": "Undo",
+  "Redo": "Redo",
+  "Cut": "Cut",
+  "Copy": "Copy",
+  "Paste": "Paste",
+  "Select all": "Select all",
+  "View": "View",
+  "Toggle full screen": "Toggle full screen",
+  "Reload": "Reload",
+  "Window": "Window",
+  "Minimize": "Minimize",
+  "About": "About",
+  "Quit": "Quit",
+  "Help": "Help",
+  "Lisk Website": "Lisk Website",
+  "Lisk Chat": "Lisk Chat",
+  "Lisk Explorer": "Lisk Explorer",
+  "Lisk Forum": "Lisk Forum",
+  "Report Issue...": "Report Issue...",
+  "What's New...": "What's New..."
+ }
```

Comment:
I'd suggest to use other name then `common.json` (e.g. `menu.json`) for this file to make it easier to distinguish from `src/locales/en/common.json` in any localization platform we will choose.